cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -1341,8 +1341,16 @@ queryResult eAcq = pure eAcq
 
 toEpochInfo :: EraHistory CardanoMode -> EpochInfo (Either Text)
 toEpochInfo (EraHistory _ interpreter) =
-  hoistEpochInfo (first (Text.pack . show ) . runExcept)
-    $ Consensus.interpreterToEpochInfo interpreter
+  hoistEpochInfo (first (Text.pack . show) . runExcept)
+    $ Consensus.interpreterToEpochInfo (Consensus.unsafeExtendSafeZone interpreter)
```

Comment:
`unsafeExtendSafeZone` should _not_ be called here, right?

Reply:
Fixed.  Thanks!